inlong-sort/sort-connectors/doris/src/main/java/org/apache/inlong/sort/doris/table/DorisStreamLoad.java

```diff
@@ -80,7 +81,7 @@ protected boolean isRedirectable(String method) {
         this.httpClient = httpClientBuilder.build();
     }
 
-    public void load(String db, String tbl, String value) throws StreamLoadException {
+    public void load(String db, String tbl, String value, SinkMetricData metricData) throws StreamLoadException {
```

Comment:
It is recommended to modify the method return instead of modifying the input parameters to support metric collection.

Reply:
accept